### PR TITLE
Ozodbek.cs ning ichiga narsa qoshildi

### LINKED ISSUE #1: Bu Issue Ozodbekga

### PR DESCRIPTION
**Pull request by Elbekovich**
Closes #1

### CODE REVIEW

**Comment by Elbekovich** [comment]
Ozodbek.cs added

**Comment by temurkodirov** [comment]
ochirvoring buni

File: PullRequesProject/Ozodbek.cs [hunk 1]
```diff
@@ -2,5 +2,9 @@
 {
     public class Ozodbek
     {
+        public static void Salom()
+        {
```
Comment (abdisamadovich) [inline PullRequesProject/Ozodbek.cs:6]: Ozodebk bu gapingizni manosi nima?

File: PullRequesProject/Ozodbek.cs [hunk 2]
```diff
@@ -2,5 +2,9 @@
...
+            Console.WriteLine("Chyo Zaaaaaaaaaaaaa?");
```
Comment (abdisamadovich) [inline PullRequesProject/Ozodbek.cs:7]: Ozodbek bu gapingizni manosi nima

**Review by abdisamadovich — CHANGES_REQUESTED** [review]
Togrlang

File: PullRequesProject/Ozodbek.cs (hunk 2 above)
Comment (Elbekovich) [reply PullRequesProject/Ozodbek.cs:7]: counter go?

Comment (Jovliyev) [reply PullRequesProject/Ozodbek.cs:7]: go goo

**Comment by Elbekovich** [comment]
a

**Comment by Elbekovich** [comment]
Closes #1

**Pull request merged**